behat/features/tabs.feature

```diff
@@ -0,0 +1,141 @@
+Feature: Book page
+  As an anonymous user
+  We need to be able to see the tabs list in the book page.
+
+  @api
+  Scenario: Visit a books page
+    Given I am an anonymous user
+    When  I visit "books/זה-ספר-בית-המדות"
+    Then I should see the text under the tab
+    | tab                         | text                                       |
+    | Book producers              | מרגליות, יהודה ליב בן אשר                  |
+    | Book structure              | על חלק דברי המוסר                          |
+    | Textual models              | מרגליות                                    |
+    # | Type of book                | Title of the book in Latin characters      |
```

Comment:
@bitamar note that in the original test there was this line, but it's no longer in that page. Not sure if ok or not, so letting you to decide.